junifer/testing/datagrabbers.py

```diff
@@ -14,8 +14,12 @@
 from ..datagrabber.base import BaseDataGrabber
 
 
-class OasisVBMTestingDatagrabber(BaseDataGrabber):
-    """Data Grabber for Oasis VBM testing data."""
+class OasisVBMTestingDataGrabber(BaseDataGrabber):
+    """Data Grabber for Oasis VBM testing data.
```

Comment:
Should there be a whitespace here in the docstring i.e. `Data Grabber` or `DataGrabber`?

Reply:
For consistency in code, maybe it's better to remove the whitespace.